fdbserver/RestoreUtil.h

```diff
@@ -58,9 +58,26 @@ struct VersionedMutation {
 	}
 };
 
+struct SampledMutation {
```

Comment:
While this struct is currently only used within a `Standalone` construct, using a reference type here inside a message that will be serialized without an explicit arena attached to it seems weird and error-prone.